homeassistant/components/lutron/__init__.py

```diff
@@ -73,7 +76,7 @@ def setup(hass, base_config):
                 hass.data[LUTRON_BUTTONS].append(
                     LutronButton(hass, keypad, button))
 
-    for component in ('light', 'cover', 'switch', 'scene'):
+    for component in ('light', 'cover', 'switch', 'scene', 'fan'):
         discovery.load_platform(hass, component, DOMAIN, None, base_config)
```

Comment:
Please pass an empty dict as discovery_info, ie the fourth argument should not be `None`.